news/2019-12-31-new-featured-artist-mylk.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "New Featured Artist: MYLK"
+date: 2019-12-31 08:00:00 +0000
+---
+
+**MYLK** joins the lineup as the last Featured Artist of 2019!
+
+![](https://assets.ppy.sh/artists/62/header.jpg)
+
+The end of the year is well and truly here and with it, our last addition to the roster for the decade. It's been a wild year of amping up the pace, and what better note to leave it on than the bubblegum blast that is **MYLK**!
+
+**MYLK**'s featured in a bunch of collaborative works across the EDM scene as a vocalist, working alongside artists like *Muzzy*, *Feint*, and *Moe Shop*.
+
+She's also an avid producer in her own right with a whopping assortment of singles and tracks to embrace, and we're delighted to announce that we have **15** brand new tracks from her ensemble for you to embrace, all guaranteed to be extremely high in calcium, sugar, and punchy EDM pop that wriggles into your earholes and refuses to leave.
+
+In a bit of a throwback to some of our earlier releases, the osu!taiko wing of the [Mappers' Guild](https://mappersguild.com/) has been hard at work creating a bunch of new sets to some of her tracks to help you end the year with a bang — literally.
+
+They're included in the preview bits below, so make sure to give them a look:
+
+### Smile
+
+[Check out the osu!taiko set](https://osu.ppy.sh/beatmapsets/1065517#taiko/2231210) by [Nardoxyribonucleic](https://osu.ppy.sh/users/876419)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/62/previews/1756.mp3" type="audio/mpeg">
+</audio>
+
+### SOAP
+
+[Check out the osu!taiko set](https://osu.ppy.sh/beatmapsets/1055618#taiko/2205809) by [tasuke912](https://osu.ppy.sh/users/2774767)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/62/previews/1755.mp3" type="audio/mpeg">
+</audio>
+
+### Bubblegum
+
+[Check out the osu!taiko set](https://osu.ppy.sh/beatmapsets/1055143#taiko/2204735) by [Cynplytholowazy](https://osu.ppy.sh/users/3901754)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/62/previews/1746.mp3" type="audio/mpeg">
+</audio>
+
+### Let You Go
+
+[Check out the osu!taiko set](https://osu.ppy.sh/beatmapsets/1053583#taiko/2201615) by [Nepuri](https://osu.ppy.sh/users/6637817)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/62/previews/1752.mp3" type="audio/mpeg">
+</audio>
+
+### Prism
+
+[Check out the osu!taiko set](https://osu.ppy.sh/beatmapsets/1051590#taiko/2197618) by [salchow](https://osu.ppy.sh/users/9738159)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/62/previews/1754.mp3" type="audio/mpeg">
+</audio>
+
+---
+
+As always, all of the above plus so much more is available right now from [**MYLK**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/62), pre-timed by the most pishi of fats and all primed and ready for you to work your mapping magic with.
```

Comment:
you have permission to kill me for trying to apply logic to a dumb joke, but just saying...
```suggestion
As always, all of the above plus so much more is available right now from [**MYLK**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/62), pre-timed by the most fat of pishis and all primed and ready for you to work your mapping magic with.
```
this may make a little more sense

Reply:
Don't worry, pishi's name is really ment to mean fat cats... sooo

Reply:
i thought of this too but figured it might be lost on people who see fatpishi instead of pishifat